autokeras/encoder.py

```diff
@@ -68,14 +73,14 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self._label_to_vec = {}
 
-    def get_state(self):
-        state = super().get_state()
-        state.update({'label_to_vec': self._label_to_vec})
-        return state
+    def get_weights(self):
```

Comment:
Why the change from `state` to `weights`? It looks like this method is returning more than just weights